pytorch_lightning/accelerators/accelerator.py

```diff
@@ -73,9 +85,9 @@ def setup(self, trainer: 'Trainer', model: LightningModule) -> None:
             trainer: the trainer instance to connect to
             model: the model to train
         """
-        self.connect_training_type_plugin(self.training_type_plugin, model)
+        self.setup_training_type_plugin(self.training_type_plugin, model)
         self.setup_optimizers(trainer)
-        self.connect_precision_plugin(self.precision_plugin)
+        self.setup_precision_plugin(self.precision_plugin)
```

Comment:
If anyone extended and made their own accelerator, this will be a breaking change so might need to handle a deprecation path here

Reply:
Do you need to rename them ?

Reply:
I don't i guess? It's just a bit concerning because if I don't rename them, `connect_training_type_plugin` will be calling `plugin.setup`, and `connect` will be calling `training_type_plugin.connect`. Just confusing function names. I think if it becomes an issue we can make this BW compatible however in most cases it seems users should be defining plugins, not accelerators.